src/config.rs

```diff
@@ -42,6 +42,10 @@ impl FrumConfig {
     pub fn aliases_dir(&self) -> std::path::PathBuf {
         ensure_dir_exists(self.base_dir().join("aliases"))
     }
+
+    pub fn temp_installations_dir(&self) -> std::path::PathBuf {
```

Comment:
Extracting this seemed useful as it's now used both in `install` as well as `clean`.